### PR TITLE
Show point information when hovering over a point.

### PR DESCRIPTION
**Pull request by maurizi**
gif

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,8 +154,10 @@
             .attr('value', function(d) { return d; })
             .html(function(d) { return d; });
         options.exit().remove();
-
-        update(indicators, range.value * 60);
```
Comment (flibbertigibbet) [inline index.html:144]: Without this, the graphs won't show.

File: index.html [hunk 2]
```diff
@@ -140,8 +154,10 @@
...
+      }).on('select', function(e) {
+          $('#info').html(infoTemplate({data: e.data.structured}));
```
Comment (flibbertigibbet) [inline index.html:164]: I'll see about modifying the `pointset` endpoint to keep the additional, non-structured properties (store name and address, for this dataset) that are in the loaded `pointset` GeoJSON, but not in what the API returns.

**Comment by flibbertigibbet** [comment]
Looks nice. How about maybe putting the interactivity info in a floating `div` in another corner and leaving the graphs in?

File: index.html (hunk 1 above)
Comment (maurizi) [inline index.html:144]: oops. fixed this.

**Comment by maurizi** [comment]
I like the suggestion to put the info in another floating div:
capture

**Comment by flibbertigibbet** [comment]
Looks great! :+1:

**Pull request merged**